pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tx3_trp"
```

Comment:
In python, package usually use `-` instead of `_`.
Lets call this one `tx3-trp`

Reply:
Perfect! I updated the name to `tx3-trp` 🙌